app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int) -> Distance:
```

Comment:
It's great that you have implemented the addition operator for the `Distance` class and that it can handle both `Distance` and `int` types. However, you should consider adding a check for negative values. If the `other` is a negative integer, the result might be a negative distance, which might not make sense in your application.